syntax_and_semantics/literals/range.md

```diff
@@ -1,14 +1,32 @@
 # Range
 
-A [Range](http://crystal-lang.org/api/Range.html) is typically constructed with a range literal:
+A [Range](http://crystal-lang.org/api/Range.html) represents an interval between two values. It is typically constructed with a range literal, consisting of two `..` or three dots:
 
-```crystal
-x..y  # an inclusive range, in mathematics: [x, y]
-x...y # an exclusive range, in mathematics: [x, y)
+* `x..y`: Two dots denote an inclusive range, including `x` and `y` and all values in between (in mathematics: `[x, y]`) .
+* `x...y`: Three dots denote an exclusive range, including `x` and all values up to but not including `y` (in mathematics: `[x, y)`).
 
-# Example:
+```cr
 (0..5).to_a # => [0, 1, 2, 3, 4, 5]
 (0...5).to_a # => [0, 1, 2, 3, 4]
 ```
 
+NOTE: Range literals are often wrapped in parenthesis, for example if it is meant to be used as the receiver of a call. `0..5.to_a` without parenthesis would be semantically equivalent to `0..(5.to_a)` because method calls and other operators have higher precedence than the range literal.
```

Comment:
I don't think `NOTE:` should be used here too because it doesn't do anything and has no markdown.
```suggestion
Note that range literals are often wrapped in parentheses, for example if it is meant to be used as the receiver of a call. `0..5.to_a` without parentheses would be semantically equivalent to `0..(5.to_a)` because method calls and other operators have higher precedence than the range literal.
```

Reply:
Yes, `NOTE` doesn't have a special syntactic meaning, but it's nevertheless a useful admonition. We're using this in other locations as well.